src/main/java/org/eclipse/pass/schema/SchemaService.java

```diff
@@ -10,23 +29,53 @@
  */
 public class SchemaService {
 
+    private PassClient client;
+    private List<String> repository_list;
```

Comment:
This class should take the repository list as an argument to a method and not store it here!